packages/base/src/InitialConfiguration.ts

```diff
@@ -39,6 +40,7 @@ let initialConfig: InitialConfig = {
 	noConflict: false, // no URL
 	formatSettings: {},
 	fetchDefaultLanguage: false,
+	fetchDefaultThemingCSSVars: true,
```

Comment:
lets make the default false like attributes, its easier to read and more logical when people want to set it